plugins/local_file/src/interface.h

```diff
@@ -0,0 +1,10 @@
+#ifndef USER_MAPPER_INTERFACE
+#define USER_MAPPER_INTERFACE
```

Comment:
Add `IRODS_HTTP_API_` to the front of these.

Reply:
Done. Add a trailing '_H' too?

Reply:
Yes please.

Reply:
Done.